README.md

```diff
@@ -10,35 +10,55 @@ This is the server component of the AeroGear metrics service. It is a RESTful AP
 * [Install the dep package manager](https://golang.github.io/dep/docs/installation.html)
 * [Install Docker and Docker Compose](https://docs.docker.com/compose/install/)
 
-## Clone and Install Dependencies
+## Setup and Build
 
-1. Clone this repository
+In Go, projects are typically kept in a [workspace](https://golang.org/doc/code.html#Workspaces) that follows a very specific architecture. Before cloning this repo, be sure you have a `GOPATH` env var that points to your workspace folder, say:
+
+```sh
+$ echo $GOPATH
+/Users/JohnDoe/workspaces/go
 ```
-go get github.com/aerogear/aerogear-metrics-api
+
+Then clone this repository by running:
+
 ```
-1. Run the following command to build binary
+git clone git@github.com:aerogear/aerogear-metrics-api.git $GOPATH/src/github.com/aerogear
+```
+
+And finally install dependencies:
+```
+make setup
```

Comment:
Setup is being part of the build. Any reason why we specify both?

Reply:
Yes, IMO if building the app is not a necessary step (which is not, unless I'm profoundly mistaken) there should be an option no to. In this case, you just have to install de dependencies and run the app, you can do it without having any binaries.